### PR TITLE
Add out argument to Int32.TryParse

### LINKED ISSUE #6930: Example usage of int.TryParse is incorrect in second paragraph
The second paragraph states:

> If you have a string, it is slightly more efficient and straightforward to call a `TryParse` method (for example, `int.TryParse("11")`). 

However, [`int.TryParse`](https://msdn.microsoft.com/en-us/library/f02979c7(v=vs.110).aspx) method expects a second parameter of Int32 type. Having known this, the sentence comes across as confusing. I had to look up the documentation to see if there wasn't an overload that allowed just one parameter.

I would suggest using `int.Parse("11")` or `int.TryParse("11", out num)` instead, or omit the inline example entirely, to avoid confusion.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f91931f7-35de-15e0-4c20-53a3fa9246ec
* Version Independent ID: d9031bdc-890a-d5bd-c1ed-d767b0e27890
* Content: [How to: Convert a String to a Number (C# Programming Guide)](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/types/how-to-convert-a-string-to-a-number)
* Content Source: [docs/csharp/programming-guide/types/how-to-convert-a-string-to-a-number.md](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/types/how-to-convert-a-string-to-a-number.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by rpetrusha**
## Add out argument to Int32.TryParse

Fixes #6930

### CODE REVIEW

**Review by jzeferino — APPROVED** [review]
LGTM

File: docs/csharp/programming-guide/types/how-to-convert-a-string-to-a-number.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ ms.assetid: 467b9979-86ee-4afd-b734-30299cda91e3
 # How to: Convert a String to a Number (C# Programming Guide)
 You can convert a [string](../../../csharp/language-reference/keywords/string.md) to a number by using methods in the <xref:System.Convert> class or by using the `TryParse` method found on the various numeric types (int, long, float, etc.).  
   
- If you have a string, it is slightly more efficient and straightforward to call a `TryParse` method (for example, `int.TryParse("11")`).  Using a `Convert` method is more useful for general objects that implement <xref:System.IConvertible>.  
+ If you have a string, it is slightly more efficient and straightforward to call a `TryParse` method (for example, `int.TryParse("11", out number)`).  Using a `Convert` method is more useful for general objects that implement <xref:System.IConvertible>.
```
Comment (BillWagner) [inline docs/csharp/programming-guide/types/how-to-convert-a-string-to-a-number.md]: nit: Should `TryParse` be an xref?

**Review by BillWagner — APPROVED** [review]
This LGTM @rpetrusha 
I did have one suggestion, unrelated to this PR, if you wanted to make that additional change. Then, :shipit: when ready.

**Comment by mairaw** [comment]
Merging this one.

**Pull request merged**